torch/csrc/jit/tensorexpr/expr.h

```diff
@@ -165,6 +165,34 @@ class Var : public ExprNode<Var> {
   std::string name_hint_;
 };
 
+// An expression to construct the underlying variable node.
```

Comment:
Why was this moved? If there is no real need in that, it's better to avoid unnecessary code movements.

Reply:
Oops, I added one constructor for `Buf `that uses `VarHandle` in prior commits so moved it ahead of `Buf`. The constructor turns out unnecessary and was removed in later commits. Forgot to move `VarHandle` back.